### ISSUE TITLE
SpecViz -- Error Displaying Widget: Model not found

### ISSUE THREAD

**Issue by atharvabagul2000** [spacetelescope/jdaviz#1494]
I am trying to load the SpecViz app with some spectrum on it but when I call Specviz
`specviz = jdaviz.Specviz()` 
`specviz.load_spectrum(spectrum)` 
`specviz.app`
 it shows me the statement
" Error Displaying Widget: Model not found" rather than displaying the app. 

I tried re-installing Jupyter Labextensions (commands mentioned below) and updating JupyterLab version but still facing the same issue.

Commands I tried in the terminal:

`jupyter labextension install @jupyter-widgets/jupyterlab-manager`
`jupyter labextension install bqplot`
`jupyter lab build`

**Desktop (please complete the following information):**
 - OS: macOS Monterey
 - Browser: Safari
 - Jupyter: (jupyterlab : 3.4.3)

**Comment by pllim** [comment]
Hi.

* Does it work in Jupyter Notebook?
* Does it work in a different browser?
* Was the Jupyter stack installed with `conda` or `pip`?

**Comment by atharvabagul2000** [comment]
> Hi.
> 
> * Does it work in Jupyter Notebook?
> * Does it work in a different browser?
> * Was the Jupyter stack installed with `conda` or `pip`?

Hello!

It does work in Jupyter Notebook. 

But why doesn't it work in Lab? I usually don't use the notebook.

**Comment by pllim** [comment]
I am not sure. I encountered this error myself only once or twice a while ago. I vaguely remember it going away after I created a fresh environment and use `conda` to install the Jupyter stack; e.g. https://jdaviz.readthedocs.io/en/latest/installation.html#common-issues

**Comment by pllim** [comment]
@atharvabagul2000 , did you ever get it to work? Did you try it on a new Python environment?

**Comment by atharvabagul2000** [comment]
> @atharvabagul2000 , did you ever get it to work? Did you try it on a new Python environment?

Hello! @pllim 

Yes, I got it to working. I am running it on Notebook only though. It still didn't work on JupyterLab and I don't know why!

I just came through another issue, so I have opened a new issue.

**Comment by pllim** [comment]
Do you still have problem with Lab after you created a new conda environment and only try to install Jdaviz in that environment?

**Comment by atharvabagul2000** [comment]
> Do you still have problem with Lab after you created a new conda environment and only try to install Jdaviz in that environment?

I will try to make a new environment and let you know real soon!
